rust/arrow/src/csv/writer.rs

```diff
@@ -97,6 +97,8 @@ pub struct Writer<W: Write> {
     has_headers: bool,
     /// The date format for date arrays
     date_format: String,
```

Comment:
It seems like these fields are somewhat confusingly named - `date_format` maybe could be called `date32_format` and `datetime_format` could be called `date64_format` -- though perhaps the root source of the confusion is that `Date32` is always in units of days and `Date64` is in units of millisecond.